src/test/java/RegistrationTests.java

```diff
@@ -0,0 +1,15 @@
+package tests;
+
+import org.testng.annotations.Test;
+import pageObjects.RegistrationPage;
+
+public class RegistrationTests extends BaseTest {
+    RegistrationPage registrationPage;
+    @Test
+    public void registrationNavigation() {
+        registrationPage = new RegistrationPage(getThreadLocal());
```

Comment:
1. call RegistrationPage class need to import correct path `import pages.RegistrationPage;`
2. remove package tests;
3. create constructor `registrationPage = new RegistrationPage(driver);`